### PR TITLE
[Data collection flow] Disable animation on previous/next

### LINKED ISSUE #2829: [Data collection] Jank when cycling between "Previous" and "Next"
Other tasks "flash" on the screen when navigating to/from a conditional task.

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #2829 

@shobhitagarwal1612 @gino-m  PTAL?

https://github.com/user-attachments/assets/3047ddc2-7d69-48fc-b304-945a63a6354c

### CODE REVIEW

**Comment by shobhitagarwal1612** [comment]
This also removes the animation. Can we keep that?

**Comment by anandwana001** [comment]
> This also removes the animation. Can we keep that?

@shobhitagarwal1612 
for this we can create custom pageTranform, but we also have another ticket where we want to have dynamic page creation, as currently the pages are getting created at dataCollection starts.

So, should we wait for that and remove this completely or for now should we go with this fix with no animation and start work on that?

**Review by shobhitagarwal1612 — APPROVED** [review]

File: ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionFragment.kt [hunk 1]
```diff
@@ -155,7 +155,7 @@ class DataCollectionFragment : AbstractFragment(), BackPressListener {
   }
 
   private fun onTaskChanged(taskPosition: TaskPosition) {
-    viewPager.currentItem = taskPosition.absoluteIndex
+    viewPager.setCurrentItem(taskPosition.absoluteIndex, false)
```
Comment (gino-m) [inline ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionFragment.kt:158]: Please add named arg to clarify what `false` refers to

**Review by gino-m — CHANGES_REQUESTED** [review]

File: ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionFragment.kt (hunk 1 above)
Comment (anandwana001) [reply ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionFragment.kt:159]: added comment.
As this is a java function, it doesn't support named argument as of kotlin.

**Review by gino-m — APPROVED** [review]

**Pull request merged**